run.py

```diff
@@ -1,7 +1,7 @@
 """docstring for run.py app import create_app."""
 from app import create_app
 
-app = create_app()
+app = create_app('development')
```

Comment:
Don't hard code the config here. just pass `config` then figure out what config it is in the lines that follow. To allow for dynamic app creation

Reply:
I have made the flask environment dynamic in commit https://github.com/queenfiona/SendITc3/pull/2/commits/c21923e331af6d27c2ab20847b1e83c2278bd3ca